tests/model/test_model.py

```diff
@@ -2033,12 +2034,13 @@ def test_toggle_stream_pinned_status(
         model.client.update_subscription_settings.assert_called_once_with(request)
 
     @pytest.mark.parametrize(
-        "narrow, event, called",
+        (
```

Comment:
Let's update `last_update_time` to `typing_event_start_time` in the commit description as I mentioned in the last review. :)